go/src/lib.rs

```diff
@@ -453,6 +453,20 @@ fn valkey_value_to_command_response(value: Value) -> RedisResult<CommandResponse
             command_response.response_type = ResponseType::Map;
             Ok(command_response)
         }
+        Value::Set(array) => {
+            let vec: Vec<CommandResponse> = array
+                .into_iter()
+                .map(|v| {
+                    valkey_value_to_command_response(v)
+                        .expect("Value couldn't be converted to CommandResponse")
```

Comment:
We should address all these `expect` in the code (i.e. make them go away)

Reply:
@eifrah-aws I agree with you and I think we can deal with it with a new PR - it would be better for the time being.